internal/pkg/values/triStateBool.go

```diff
@@ -28,3 +28,6 @@ func (t *triStateBool) String() string {
 	}
 	return fmt.Sprint(t.value)
 }
+
+// IsBoolFlag makes the argument not have a value
+func (b *triStateBool) IsBoolFlag() bool { return true }
```

Comment:
What do you do with this? I don't see it being used anywhere.

Reply:
This makes the triStateBool type implement the flag.boolFlag interface which allows the cli arguments to have no value eg.: "--explicit-include" instead of "--explicit-include=true"

Reply:
I did notice that the comment on the function didn't explain this very well, so i changed that. Its function should now be clearer